invenio_stats/contrib/events.py

```diff
@@ -23,12 +23,25 @@
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
 """Registration of contrib events."""
+from invenio_search import current_search_client
+
 from invenio_stats.indexer import EventsIndexer
+from invenio_stats.tasks import StatAggregator
 
 
-def register():
+def register_events():
     """Register sample events."""
     return [dict(event_type='file-download',
                  processor=EventsIndexer),
             dict(event_type='record-view',
                  processor=EventsIndexer)]
+
+
+def register_aggregations():
+    """Register sample aggregations."""
+    return [dict(event_type='file-download',
```

Comment:
The name should not be the name of an event type. We could have multiple aggregations based on the same events.